tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/call_settings.py

```diff
@@ -239,8 +239,8 @@ def tap_call_barring(self):
         Wait(self.marionette).until(expected.element_displayed(*self._call_settings_query_locator))
 
     def wait_until_call_barring_info_received(self):
-        Wait(self.marionette, timeout=60).until(
-            expected.element_displayed(*self._call_barring_status_disabled_text_locator))
+        Wait(self.marionette, timeout=120, interval=1).until(
```

Comment:
not on local device, but i learned that on jenkins it takes much slower.  looks like it's taking around 60 seconds, so I doubled it.  The script that takes less than 2 min locally, it takes 6 minutes on jenkins according to the timestamp, for some reason.